lib/file-list.js

```diff
@@ -16,127 +13,54 @@ const helper = require('./helper')
 const log = require('./logger').create('watcher')
 const createPatternObject = require('./config').createPatternObject
 
-// Constants
-// ---------
-
-const GLOB_OPTS = {
-  cwd: '/',
-  follow: true,
-  nodir: true,
-  sync: true
-}
-
-// Helper Functions
-// ----------------
-
-const byPath = (a, b) => {
+function byPath (a, b) {
   if (a.path > b.path) return 1
   if (a.path < b.path) return -1
 
   return 0
 }
 
-/**
```

Comment:
This is and several comments below seem to have value as they explain implementation details. Why do you remove them?

Reply:
I don't think these comments provide any value. Code should explain implementation details itself.